### PR TITLE
Add Osmo

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://sourceforge.net/p/osmo-pim/feature-requests/156/
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://sourceforge.net/p/osmo-pim/patches/25/

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build net.sourceforge.osmo

File: net.sourceforge.osmo.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  - name: libmcrypt
+    buildsystem: autotools
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/mcrypt/files/Libmcrypt/2.5.8/libmcrypt-2.5.8.tar.gz
+        sha256: e4eb6c074bbab168ac47b947c195ff8cef9d51a211cdd18ca9c9ef34d27a373e
+      - type: shell
+        commands: autoconf -f -i
```
Comment (hfiguiere) [inline net.sourceforge.osmo.yml:87]:
```suggestion
        commands: 
          - cp -p /usr/share/automake-*/config.{sub,guess} .
          - autoconf -f -i
```

Comment (hfiguiere) [reply net.sourceforge.osmo.yml:87]: that's to fix the build failure on aarch64

Comment (Mailaender) [reply net.sourceforge.osmo.yml:87]: Thanks

**Comment by Mailaender** [comment]
bot, build net.sourceforge.osmo

**Comment by barthalion** [comment]
One of libraries ships too old autotools files to be built on aarch64. Search for `config.guess` in the Flathub org for examples how to fix it.

**Comment by hfiguiere** [comment]
> One of libraries ships too old autotools files to be built on aarch64. Search for `config.guess` in the Flathub org for examples how to fix it.

https://github.com/flathub/flathub/pull/3441#discussion_r956485781

But it might be in the wrong directory. I didn't check.

**Comment by Mailaender** [comment]
The files to overwrite are there. Not sure why it fails. I also don't have hardware to test.

**Comment by barthalion** [comment]
bot, build net.sourceforge.osmo

**Comment by barthalion** [comment]
bot, build net.sourceforge.osmo

**Pull request closed without merging**